google-cloud-storage/lib/google/cloud/storage/service.rb

```diff
@@ -224,27 +226,29 @@ def copy_file source_bucket_name, source_file_path,
         def rewrite_file source_bucket_name, source_file_path,
```

Comment:
I might be missing something, but isn't the implementation of this method exactly the same as `copy_file`, above? Sorry I just can't see the difference.